packages/transactions/src/coders/transaction.test.ts

```diff
@@ -319,7 +319,7 @@ describe('TransactionCoder', () => {
     const encoded = hexlify(new TransactionCoder().encode(transaction));
 
     expect(encoded).toEqual(
-      '0x000000000000000300000000000000000000000000000001d5579c46dfcc7f18207013e65b44e4cb4e2c2298f4ac457ba8f82743f31e930b000000000000000500000000000000000000000000000000000000000000000000000000000003e800000000000003e8'
+      '0x000000000000000300000000000000000000000000000001d5579c46dfcc7f18207013e65b44e4cb4e2c2298f4ac457ba8f82743f31e930b0000000500000000000000000000000003e800000000000003e8'
```

Comment:
All these values should be extracted to named constants otherwise it's difficult to determine why this should be the expected result as opposed to some other _similar_ looking string

Reply:
This is a nit.

We can go without it.